aya/src/programs/mod.rs

```diff
@@ -797,6 +819,57 @@ impl_fd!(
     CgroupDevice,
 );
 
+/// Defines the [`Program`] types which support the kernel's
+/// [generic multi-prog API](https://github.com/torvalds/linux/commit/053c8e1f235dc3f69d13375b32f4209228e1cb96).
+///
+/// # Minimum kernel version
+///
+/// The minimum kernel version required to use this feature is 6.6.0.
+pub trait MultiProgProgram {
```

Comment:
MultiProgProgram kinda stutters... maybe MultiProgram?

Reply:
I agree, and MultiProgram sounds better.  Done.